test/cpu_fusion.cpp

```diff
@@ -1125,3 +1127,64 @@ TEST(cpu_fusion, weight_fusion)
                   new_convert_layout->get_argument(0)),
               cvt_lt_conv);
 }
+
+TEST(cpu_fusion, sigmoid_multiply_fusion_compute)
+{
+    auto backend = runtime::Backend::create("CPU");
+
+    const size_t w = 2;
```

Comment:
`w, h` seem a bit redundant here. `SigmoidMultiply` doesn't seem to care what those dimensions mean.

Reply:
fixed.